src/player.py

```diff
@@ -1,2 +1,55 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
```

Comment:
Looks great 🔥 . I really like the extra attributes you added to the player class to make this more like an actual game.